server/src/main/java/com/openelements/benchscape/server/store/endpoints/BenchmarkEnvironmentEndpoint.java

```diff
@@ -111,10 +112,13 @@ public List<Environment> findByQuery(@RequestParam(required = false) String name
             @RequestParam(required = false) String osVersion,
             @RequestParam(required = false) String jvmVersion,
             @RequestParam(required = false) String jvmName,
+                                         @RequestParam(required = false) OperationSystem osFamily,
+                                         @RequestParam(required = false) String systemMemoryReadable,
```

Comment:
Fine for an initial version. I assume we need to change that in future: I do not see a use case where system memory is not defined in GB. So maybe we do an integer systemMemoryInGb here. That can simply be converted in the `Long systemMemory` that is used in `EnvironmentQuery`.